pom.xml

```diff
@@ -3220,9 +3217,34 @@
 
     <profile>
       <id>scala-2.12</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
       <build>
         <pluginManagement>
           <plugins>
+            <plugin>
+              <groupId>net.alchim31.maven</groupId>
+              <artifactId>scala-maven-plugin</artifactId>
+              <version>4.4.0</version>
+              <configuration>
+                <args combine.children="append">
+                    <arg>-unchecked</arg>
+                    <arg>-Xfatal-warnings</arg>
+                    <arg>-deprecation</arg>
+                    <arg>-Ywarn-unused-import</arg>
+                    <arg>-P:silencer:globalFilters=.*deprecated.*</arg>
+                    <arg>-feature</arg>
+                </args>
+                <compilerPlugins>
+                  <compilerPlugin>
+                    <groupId>com.github.ghik</groupId>
+                    <artifactId>silencer-plugin_${scala.version}</artifactId>
+                    <version>1.6.0</version>
```

Comment:
Missing `com.github.ghik:silencer-plugin_2.12.10:1.7.1` in Maven central repository